samples/subsys/usb/cdc_acm/README.rst

```diff
@@ -0,0 +1,63 @@
+.. _usb_cdc_acm:
+
+USB CDC ACM Sample Application
+####################################
+
+Overview
+********
+
+Sample app for USB CDC ACM driver. The received data is echoed back
+to the serial port provided by Zephyr CDC ACM driver.
+
+Requirements
+************
+
+This project requires an USB device driver,
+which is available for the :ref:`arduino_101` and :ref:`frdm_k64f` boards.
+
+Building and Running
+********************
+
+Plug the board into a host device, for example, a PC running Linux.
+The board will be detected as a CDC ACM device.
+
+.. code-block:: console
+
+   usb 9-1: new full-speed USB device number 112 using uhci_hcd
+   usb 9-1: New USB device found, idVendor=8086, idProduct=f8a1
+   usb 9-1: New USB device strings: Mfr=1, Product=2, SerialNumber=3
+   usb 9-1: Product: CDC-ACM
+   usb 9-1: Manufacturer: Intel
+   usb 9-1: SerialNumber: 00.01
+   cdc_acm 9-1:1.0: ttyACM1: USB ACM device
+
+The app prints on serial output, used for the console:
+
+.. code-block:: console
+
+   Wait for DTR
+
+Open a serial port emulator, for example minicom
```

Comment:
no, the sample waits for DTR, it appears after the minicom opens the device provided by the Zephyr.